ni_measurementlink_service/session_management/_reservation.py

```diff
@@ -160,7 +160,30 @@ class _ConnectionKey(NamedTuple):
     instrument_type_id: str
 
 
-class BaseReservation(abc.ABC):
+class _BaseSessionContainer(abc.ABC):
+    """Manages Session Management Client."""
+
+    def __init__(
+        self,
+        session_manager: SessionManagementClient,
```

Comment:
This name is outdated. We renamed the session manager service to the session management service prior to the 1st MeasurementLink release.

Reply:
Updated as `session_management_client`.